packages/core/__tests__/notes.test.ts

```diff
@@ -166,11 +166,11 @@ test("note title should allow trailing space", () =>
     }
   ));
 
-test("note title should not allow newlines", () =>
+test("note title should allow newlines", () =>
   noteTest({ title: "Hello\nhello", content: TEST_NOTE.content }).then(
     async ({ db, id }) => {
       const note = await db.notes.note(id);
-      expect(note?.title).toBe("Hello hello");
+      expect(note?.title).toBe("Hello\nhello");
```

Comment:
We don't allow newlines in titles. Why is the test being changed here?

Reply:
This PR adds support for multilines which includes newlines as well

Reply:
We have to strip newlines. Multilines basically means wrapping not actual multiple lines.